packages/manager/modules/telecom-spare/src/modems/modal/modems-modal.controller.js

```diff
@@ -0,0 +1,144 @@
+export default class {
+  /* @ngInject */
+  constructor($uibModalInstance, OvhApiXdsl, params) {
+    this.$uibModalInstance = $uibModalInstance;
```

Comment:
You should prefer using a modal with a state 
You can find various examples especially in pci module (e.g : https://github.com/ovh-ux/manager/blob/master/packages/manager/modules/pci/src/projects/project/instances/resume/resume.routing.js)
https://github.com/ovh-ux/manager/tree/master/packages/components/ng-ui-router-layout will help you to do so